Mathlib/Computability/TMToPartrec.lean

```diff
@@ -1240,9 +1240,6 @@ def K'.elim (a b c d : List Γ') : K' → List Γ'
   | K'.stack => d
 #align turing.partrec_to_TM2.K'.elim Turing.PartrecToTM2.K'.elim
 
-/- Porting note: The equation lemma of `elim` simplifies to `match` structures. To prevent this,
```

Comment:
The first sentence of this porting note is still true, so I don't think we should remove it.

Reply:
Given we have two other porting notes in this file about equation lemmas, I don't think we should remove this one.

Reply:
🤷 

I've restored it, no longer as a porting note.